ovirt/resource_ovirt_vm.go

```diff
@@ -704,6 +764,24 @@ func ovirtRemoveGraphicsConsoles(vmID string, meta interface{}) error {
 	return nil
 }
 
+func ovirtUpdateAutoPinning(vmID string, autoPinningPolicy ovirtsdk4.AutoPinningPolicy, meta interface{}) error {
```

Comment:
I think this function just sets the "optimizeCpuSettings", Maybe you should change the name to make it clearer and reflect what the function is doing

Reply:
It sets the auto pinning policy, how about: `ovirtSetAutoPinningPolicy`?